core/WORKSPACE

```diff
@@ -1,2 +1,67 @@
-# only temporary for compatibility with `WORKSPACE` setup
-# TODO: remove when migration to `bzlmod` is complete
\ No newline at end of file
+# NOTE: only temporary for compatibility with `WORKSPACE` setup!
+# TODO: remove when migration to `bzlmod` is completelocal_repository
+
```

Comment:
Best add a comment to explain that the following are only dev-dependencies needed for stardoc.